Tests/Model/SimpleParent.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Codete\FormGeneratorBundle\Tests\Model;
+
+use Codete\FormGeneratorBundle\Annotations as Form;
+
+/**
+ * @Form\Form(
+ *   default = { "person", "named", "anonymous", "employee" }
+ * )
+ */
+class SimpleParent
+{
+    public $id;
+    
+    /**
+     * @Form\Embed(
+     *   class = "Codete\FormGeneratorBundle\Tests\Model\Person",
+     * )
+     */
+    public $person;
+
+    /**
+     * @Form\Embed(
+     *   class = "Codete\FormGeneratorBundle\Tests\Model\Person",
+     * )
+     */
+    public $named;
+
+    /**
+     * @Form\Embed(
+     *   class = "Codete\FormGeneratorBundle\Tests\Model\Person",
+     *   context = {
+     *     "no_photo" = "true"
```

Comment:
I know these are only tests, but `"true"` looks weird, why not `true` that would be a boolean?